elements/pfe-dropdown/pfe-dropdown.ts

```diff
@@ -249,7 +265,11 @@ export class PfeDropdown extends LitElement {
   }
 
   /** modify DOM if custom options are passed in an array */
-  private async _modifyDOM(options: PfeDropdownOption[]) {
+  private async _modifyDOM(options: PfeDropdownOption[], clean: Boolean = true) {
```

Comment:
```suggestion
  private async _modifyDOM(options: PfeDropdownOption[], clean = true) {
```

`clean` probably isn't the `Boolean` constructor, and anyways for boolean default params, TS can infer the type